app/src/ui/discard-changes/discard-changes-dialog.tsx

```diff
@@ -84,7 +85,10 @@ export class DiscardChanges extends React.Component<
         <DialogFooter>
           <ButtonGroup destructive={true}>
             <Button type="submit">Cancel</Button>
```

Comment:
I think we need to add the following attribute to the _Cancel_ button: `disabled={this.state.isDiscardingChanges}`.